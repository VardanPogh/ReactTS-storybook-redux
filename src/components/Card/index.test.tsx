import React from 'react'
import { render } from '@testing-library/react'

import MediaCard from '.'

describe('MediaCard Test', () => {
  test('Loads MediaCard', () => {
    const myMock = jest.fn()
    const { container } = render(
      <MediaCard
        variant="withButton"
        title="Get a Demo"
        subTitle="Qualified engineers, scientists and PhDs with hands-on experience"
        onClick={myMock}
      />
    )
    expect(container).toMatchSnapshot()
  })
})
