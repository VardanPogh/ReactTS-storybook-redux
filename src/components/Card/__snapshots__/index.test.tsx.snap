// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MediaCard Test Loads MediaCard 1`] = `
<div>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      class="MuiCardContent-root"
    >
      <h2
        class="MuiTypography-root makeStyles-title-4 MuiTypography-h5 MuiTypography-gutterBottom"
      >
        Get a Demo
      </h2>
      <p
        class="MuiTypography-root makeStyles-subTitle-6 MuiTypography-body2 MuiTypography-colorTextSecondary"
      >
        Qualified engineers, scientists and PhDs with hands-on experience
      </p>
    </div>
    <div
      class="MuiCardActions-root MuiCardActions-spacing"
    >
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-contained-8"
        tabindex="0"
        type="button"
      >
        <span
          class="MuiButton-label"
        >
          undefined
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
